src/Umbraco.Web.BackOffice/Mapping/MemberMapDefinition.cs

```diff
@@ -0,0 +1,104 @@
+using Umbraco.Web.Models.ContentEditing;
+using Umbraco.Web.Trees;
+
+namespace Umbraco.Web.Models.Mapping
+{
+    /// <summary>
+    /// Declares model mappings for members.
+    /// </summary>
+    public class MemberMapDefinition : IMapDefinition
+    {
+        private readonly CommonMapper _commonMapper;
+        private readonly CommonTreeNodeMapper _commonTreeNodeMapper;
+        private readonly MemberTabsAndPropertiesMapper _tabsAndPropertiesMapper;
+        private readonly IHttpContextAccessor _httpContextAccessor;
+
+        public MemberMapDefinition(CommonMapper commonMapper, CommonTreeNodeMapper commonTreeNodeMapper, MemberTabsAndPropertiesMapper tabsAndPropertiesMapper, IHttpContextAccessor httpContextAccessor)
+        {
+            _commonMapper = commonMapper;
+            _commonTreeNodeMapper = commonTreeNodeMapper;
+
+            _tabsAndPropertiesMapper = tabsAndPropertiesMapper;
+            _httpContextAccessor = httpContextAccessor;
+        }
+
+        public void DefineMaps(UmbracoMapper mapper)
+        {
+            mapper.Define<IMember, MemberDisplay>((source, context) => new MemberDisplay(), Map);
+            mapper.Define<IMember, MemberBasic>((source, context) => new MemberBasic(), Map);
+            mapper.Define<IMemberGroup, MemberGroupDisplay>((source, context) => new MemberGroupDisplay(), Map);
+            mapper.Define<IMember, ContentPropertyCollectionDto>((source, context) => new ContentPropertyCollectionDto(), Map);
+        }
+
+        // Umbraco.Code.MapAll -Properties -Errors -Edited -Updater -Alias -IsChildOfListView
+        // Umbraco.Code.MapAll -Trashed -IsContainer -VariesByCulture
+        private void Map(IMember source, MemberDisplay target, MapperContext context)
+        {
+            target.ContentApps = _commonMapper.GetContentApps(source);
+            target.ContentTypeId = source.ContentType.Id;
+            target.ContentTypeAlias = source.ContentType.Alias;
+            target.ContentTypeName = source.ContentType.Name;
+            target.CreateDate = source.CreateDate;
+            target.Email = source.Email;
+            target.Icon = source.ContentType.Icon;
+            target.Id = source.Id;
+            target.Key = source.Key;
+            target.Name = source.Name;
+            target.Owner = _commonMapper.GetOwner(source, context);
+            target.ParentId = source.ParentId;
+            target.Path = source.Path;
+            target.SortOrder = source.SortOrder;
+            target.State = null;
+            target.Tabs = _tabsAndPropertiesMapper.Map(source, context);
+            target.TreeNodeUrl = _commonTreeNodeMapper.GetTreeNodeUrl<MemberTreeController>(source);
+            target.Udi = Udi.Create(Constants.UdiEntityType.Member, source.Key);
+            target.UpdateDate = source.UpdateDate;
+            target.Username = source.Username;
+        }
+
+        // Umbraco.Code.MapAll -Trashed -Edited -Updater -Alias -VariesByCulture
```

Comment:
Since I'm not overly sure what these comments mean, maybe a few extra words to explain the context for those who don't know about Umbraco.Code.MapAll?

Reply:
These comments are not changed in this PR, 

But there is a roslyn compiler ensuring errors if not all properties are mapped, or excluded in the comment as in this example.